### PR TITLE
refactored app.js

### PR DESCRIPTION
**Pull request by hermesxgjini**
encapsulated most of timer logic into a seperate class in the app.js file

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -1,151 +1,159 @@
...
+
+    stopTimer() {
+        clearInterval(this.pomodoroIntervalId);
+    }
+
+    resetTimer(selector) {
+        this.minutes = this.initialMinutes;
+        this.seconds = this.initialSeconds;
+        clearInterval(this.pomodoroIntervalId);
+        const lol =
```
Comment (amitmerchant1990) [inline js/app.js]: Hi @hermesxgjini Can you please change the variable name to something significant?

Comment (hermesxgjini) [inline js/app.js]: oh sorry, it was testing, forgot to remove it

**Comment by amitmerchant1990** [comment]
Thanks @hermesxgjini This looks good now. Good work! 🎉

**Comment by hermesxgjini** [comment]
thanks

**Pull request merged**